pandas/tests/scalar/test_nat.py

```diff
@@ -141,9 +141,10 @@ def test_nat_iso_format(get_nat):
 
 
 @pytest.mark.parametrize("klass,expected", [
-    (Timestamp, ["freqstr", "normalize", "to_julian_date", "to_period", "tz"]),
-    (Timedelta, ["components", "delta", "is_populated", "to_pytimedelta",
-                 "to_timedelta64", "view"])
+    (Timestamp, ["freqstr", "normalize", "to_julian_date", "to_numpy",
+                 "to_period", "tz"]),
+    (Timedelta, ["components", "delta", "is_populated", "to_numpy",
+                 "to_pytimedelta", "to_timedelta64", "view"])
 ])
 def test_missing_public_nat_methods(klass, expected):
```

Comment:
Why not implement to_numpy for NaT?

Reply:
It should be. Will fix that and revert the test.